postgres/init.sql

```diff
@@ -124,3 +124,13 @@ CREATE TABLE NFTs (
   blockNumber integer NOT NULL,
   minter char(64) NOT NULL
 );
+
+CREATE TABLE NFTLikes (
+  id SERIAL PRIMARY KEY,
```

Comment:
instead of having an id, you can do something like `PRIMARY KEY (nftKey, liker)`. That would probably be the best way to prevent double likes.

Reply:
but a liker can like multiple different NFT, while an NFT can have different likers